third-party/crowdsec/src/test/java/org/apache/james/CrowdsecExtension.java

```diff
@@ -40,7 +43,8 @@ public CrowdsecExtension() {
             .withCreateContainerCmdModifier(cmd -> cmd.withName("james-crowdsec-test-" + UUID.randomUUID()))
             .withExposedPorts(CROWDSEC_PORT)
             .withStartupTimeout(STARTUP_TIMEOUT)
-            .waitingFor(new HostPortWaitStrategy().withRateLimiter(RateLimiters.TWENTIES_PER_SECOND));;
+            .waitingFor(new HostPortWaitStrategy().withRateLimiter(RateLimiters.TWENTIES_PER_SECOND));
+        this.crowdsecContainer.setPortBindings(ImmutableList.of("127.0.0.1:" + EXPOSED_PORT + ":" + CROWDSEC_PORT));
```

Comment:
Let this be a random exposed port as other test containers rather than rely on a strict exposed port:
- concurrent test can be conflict on a port
- we do not know for sure if the 8082 port is always available on the host machine e.g. Apache CI (can run other project builds as well).